src/app/cases/styles.tsx

```diff
@@ -16,7 +16,7 @@ export const FiltersContainer = styled.div`
 
```

Comment:
Design Review (placed here to allow for threading)
---

I think we want the center divider to be centered between the two and use the mid grey color if possible!

Current | Figma
:--------:|:-------:
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/a861c058-777d-4ac5-8531-c2ae7eaa9e94) | ![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/23e61aa4-7d6c-4fdc-b01f-a1d0ecbefd9c)

Reply:
updated spacing + color 👍